idaplugin/rematch/instances/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .function import EmptyFunctionInstance, FunctionInstance
 from .data import EmptyDataInstance, DataInstance
 
-__all__ = [EmptyFunctionInstance, FunctionInstance,
-           EmptyDataInstance, DataInstance]
+__all__ = ["EmptyFunctionInstance", "FunctionInstance",
+           "EmptyDataInstance", "DataInstance"]
```

Comment:
Surprised me as well but apparently `__all__` should be a list of strings, although for some reason objects worked fine.